package.json

```diff
@@ -21,6 +21,7 @@
   ],
   "dependencies": {
     "rimraf": "3.0.2",
+    "smol-toml": "^1.4.2",
```

Comment:
Should this be here? If not, can you remove, and also, can you make it a precise version?